src/components/Summary/Recap.js

```diff
@@ -1,15 +1,46 @@
 import React, { Component } from 'react';
 import './index.css';
+import { connect } from 'react-redux';
 
 
+// only shown after a learning session
 class Recap extends Component {
   render() {
     return (
       <div className="Recap">
-        Only shown after a learning session
+        <div className="leftCol">
+          <h2>Recap</h2>
+        </div>
+        <div className="rightCol">
+          You&#39;re doing great! Some feedback on this session:
+
+          <ul>
+            { this.props.bath.sentences.map( (sentence) => {
+              return (
+                <li
+                  key={ sentence.lexemeId }
+                  className={ sentence.success ? 'correct' : 'wrong' } >
+                    { sentence.task.q[0] + ' ' }
+                    { sentence.success
+                      ? ''
+                      : <span className="wrongAnswer">{ sentence.answer }</span>
+                    }
+                    <span className="correctAnswer">{ ' ' + sentence.task.a }</span>
+                    { ' ' + sentence.task.q[1] }
+                </li>
+              )
+            } )}
+          </ul>
+
+        </div>
       </div>
     );
   }
 }
 
-export default Recap;
+
+const mapStateToProps = (state, props) => {
```

Comment:
If you are not using the props don't pass it ;)
`const mapStateToProps = ({ bath }) => ({ bath});`